lib/iris/tests/results/COLPEX/small_colpex_theta_p_alt.cml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <cubes xmlns="urn:x-iris:cubeml-0.2">
-  <cube standard_name="air_potential_temperature" units="K">
+  <cube core-dtype="float32" dtype="float32" fill_value="-1.07374e+09" standard_name="air_potential_temperature" units="K">
```

Comment:
I'm personally not a fan of the new bits showing up at the _start_ of the tag rather than the end. To me it feels like the `standard_name` and `units` are more important than these new bits, so if it's easy to do can we shift these new bits to the end of the tag?